code/modules/vending/_vending.dm

```diff
@@ -1007,6 +1008,9 @@ GLOBAL_LIST_EMPTY(vending_products)
 
 	tilt(L)
 
+/obj/machinery/vending/attack_tk_grab(mob/user)
+	to_chat(user, "<span class='warning'>[src] seems to resist your mental grasp!</span>")
```

Comment:
Could this just be a no-op? Maybe with a comment on it for explanation as to _why_ it's a no-op. Using TK on something and having nothing happen is much more immersive than using TK on something and having an OOC message appear.

Reply:
Failing silently with no indication of why is much worse than showing a simple (not at all OOC) message saying you can't do it and moving on with your life